src/Components/Join-now.js

```diff
@@ -78,59 +100,84 @@ export default function JoinNow() {
               <Stepper className="stepper" />
             </p>
           </div>
-            <div className="content-flex">
-              <p className="join-message">
-                <strong>Join SWECC </strong>
-              </p>
-              <p className="join-message"></p>
-              <div className="stepper-wrapper">
-                <div className={`stepper-item ${count > 0 ? "completed" : count === 0 ? "active" : ""}`}>
-                  <div className="step-counter">1</div>
-                  <div className="step-name">Join our discord</div>
-                </div>
-                <div className={`stepper-item ${count > 1 ? "completed" : count === 1 ? "active" : ""}`}>
-                  <div className="step-counter">2</div>
-                  <div className="step-name">Join our mailing list</div>
-                </div>
-                <div className={`stepper-item ${count > 2 ? "completed" : count === 2 ? "active" : ""}`}>
-                  <div className="step-counter">3</div>
-                  <div className="step-name">Attend an event</div>
-                </div>
+          <div className="content-flex">
+            <p className="join-message">
+              <strong>Join SWECC </strong>
+            </p>
+            <p className="join-message"></p>
+            <div className="stepper-wrapper">
+              <div
+                className={`stepper-item ${count > 0 ? "completed" : count === 0 ? "active" : ""}`}
+              >
+                <div className="step-counter">1</div>
+                <div className="step-name">Join our discord</div>
+              </div>
+              <div
+                className={`stepper-item ${count > 1 ? "completed" : count === 1 ? "active" : ""}`}
+              >
+                <div className="step-counter">2</div>
+                <div className="step-name">Join our mailing list</div>
+              </div>
+              <div
+                className={`stepper-item ${count > 2 ? "completed" : count === 2 ? "active" : ""}`}
```

Comment:
let's create a helper function for this where we can pass in a value and it returns completed, active or empty string then we don't have to have this long thing of code 3 times.